raiden/network/transport/matrix/utils.py

```diff
@@ -342,9 +342,11 @@ def query_capabilities_for_user_id(self, user_id: str) -> PeerCapabilities:
         """ This pulls the `avatar_url` for a given user/user_id and parses the capabilities.  """
         try:
             user: User = self._client.get_user(user_id)
-            avatar_url = user.get_avatar_url()
+            avatar_url = user.api.get_avatar_url(user.user_id)
             if avatar_url is not None:
                 return PeerCapabilities(deserialize_capabilities(avatar_url))
+        except ValueError as e:
+            log.error("Error when fetching avatar url", exception=e, user_id=user_id)
```

Comment:
Can a `ValueError` even happen anymore when calling the api method directly?

Reply:
I can not, unless the API call implementation changes. Removed the catch.